### PR TITLE
Move sending process to a background thread. Closes issue #5

### PR DESCRIPTION
**Pull request by AparaV**
## Changes
* Created a background worker with a thread and delegated sending SMS to the worker.
* Created a new `sending` view to which the user is redirected to. This view makes AJAX calls every 2 seconds to check for completion of thread. Upon completion, user is redirected to `successfully_sent` view.
* Updated README.md to warn users of the potential to be blacklisted by phone carriers. Other minor edits to setup instructions.

### Files added
* `mercurysms/templates/sending.html`
* `mercurysms/worker.py`

### Files modified
* `.gitignore`
* `README.md`
* `mercurysms/templates/base.html`
* `mercurysms/urls.py`
* ` mercurysms/views.py`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,7 +75,7 @@ A list is a column in the mentioned Google Sheets. It is identified by each firs
 
 - Open root route. Ex: http://localhost:8000
 - Login with user (you can login with the super user you created in your first steps)
-- Send messages right away. Let's SPAM those users :D
+- Send messages right away.
```
Comment (casassg) [inline README.md:78]: 😄

File: mercurysms/templates/base.html [hunk 2]
```diff
@@ -5,6 +5,7 @@
     <meta charset="UTF-8">
     <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css">
     <script src="https://code.jquery.com/jquery-3.1.1.slim.min.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script>
```
Comment (casassg) [inline mercurysms/templates/base.html:8]: why this? there's already a jquery (line above)

**Comment by AparaV** [comment]
Bump @casassg

**Review by casassg — APPROVED** [review]
let's try it 😄

File: mercurysms/templates/base.html (hunk 2 above)
Comment (AparaV) [reply mercurysms/templates/base.html:8]: It said `jquery-slim` and I wasn't sure if they were the same thing 🤷‍♂️

**Pull request merged**